aidial_adapter_vertexai/chat/static_tools.py

```diff
@@ -27,21 +27,62 @@ def parse_gemini_tools(
     ) -> List[GeminiTool] | None: ...
 
 
+class DynamicRetrievalConfig(BaseModel):
+    mode: Literal["MODE_DYNAMIC", "MODE_UNSPECIFIED"] | None = None
```

Comment:
Should we use aliases for camel case for the fields?

Because I see in the REST documentation the following request:

```json
{
  "contents": [{
    "role": "user",
    "parts": [{
      "text": "TEXT"
    }]
  }],
  "tools": [{
    "googleSearchRetrieval": {
      "dynamicRetrievalConfig": {
        "mode": "MODE_DYNAMIC",
        "dynamicThreshold": DYNAMIC_THRESHOLD
      }
    }
  }],
  "model": "projects/PROJECT_ID/locations/LOCATION/publishers/google/models/MODEL_ID"
}
```

And I'm sure DIAL users will look at this documentation and try to configure the tool this way, and they will be surprised by 400 error.

Reply:
Fixed, added aliases